rails-snow-queen/app/views/quotes/new.html.erb

```diff
@@ -1,17 +1,25 @@
-<h1>Quote calculator</h1>
+<div class="d-flex row">
+   <div id="panel" class="col-4 p-5">
+      <h1>Quote calculator</h1>
+      <p>Get a free quote for a salting or snow clearing service</p>
 
-<p>Get a free quote for a salting or snow clearing service</p>
+      <form action="" id="AddressSearch">
+         <label for="address">Search your location:</label><br>
+         <input id="address" type="textbox" placeholder="123 Street Rd, Port Moody" size="40">
+         <input id="submit" type="submit" value="Find" class="btn btn-light btn-sm">
+      </form>
+      <div class="hidden address" id="displayAddress">
+      </div>
+      <div class="hidden">
+         <h4 id="calculatedArea" class="m-3 mt-5">Selected area: _______ sq ft</h4>
+         <h3 id="totalDue" class="text-right p-3">Total    $____.__</h3>
```

Comment:
Don't use `h#` for styling. Style via CSS. Also, you might want to look into HTMLs `<dl>` tag for semantic purity :stuck_out_tongue:. Also, have the JS set only the numeric part and not worry about the labels.
```html
<dl>
  <dt>Selected area:</dt>
  <dd><span id="calculatedArea"></span> sq ft</dd>
  <dt>Total:</dt>
  <dd>$<span id="totalDue"></span></dd>
</dl>
```

Reply:
Done!